src/js/components/common/dialogs/physicaldeviceonboarding.js

```diff
@@ -100,6 +100,18 @@ export class PhysicalDeviceOnboarding extends React.Component {
               </p>
             </div>
           </ReactTooltip>
+          {hasConvertedImage && (
+            <div className="margin-top">
+              <p>
+                We prepared an image, fully integrated with Mender for you to start with. You can find it on our{' '}
+                <a href={`https://docs.mender.io/${docsVersion}downloads#disk-images`} target="_blank">
+                  downloads page
+                </a>{' '}
+                and once you&apos;re done flashing you can go ahead and proceed to the next step.
+              </p>
+              <p>If you already have an image running, you can proceed but some limitations to the kind of updates you can install apply.</p>
```

Comment:
I would rephrase to something that explicitly points to the limitation. Something similar to:
`If you already have an image running, you can proceed with this tutorial but you will not be able to do full system updates later on.`